awx/main/wsbroadcast.py

```diff
@@ -32,7 +32,13 @@ def unwrap_broadcast_msg(payload: dict):
 
 def get_broadcast_hosts():
     Instance = apps.get_model('main', 'Instance')
-    instances = Instance.objects.exclude(hostname=Instance.objects.me().hostname).exclude(version__startswith='ansible-runner-').order_by('hostname').values('hostname', 'ip_address').distinct()
+    instances = (
+        Instance.objects.exclude(hostname=Instance.objects.me().hostname)
+        .exclude(node_type='execution')
+        .order_by('hostname')
+        .values('hostname', 'ip_address')
+        .distinct()
+    )
```

Comment:
To address CI failures.

Don't blame me for the `(` usage, this is from black.